tests/regression/lp-1898038/task.yaml

```diff
@@ -0,0 +1,23 @@
+summary: docker-support and multipass-support should mount base /etc/apparmor.d
+prepare: |
+    tests.cleanup prepare
```

Comment:
Thank you for using the helper for cleanup! Removing files from the test directory is good practice but is not strictly required since there are higher-level cleanups performed around each test. Still it's fine to keep this as it's technically correct.

Reply:
Ah ok I just copied this from another task.yaml... let me know if you want it reworked to be more idiomatic.

Reply:
That's fine. I just wanted to tell you more, there's nothing I would change here yet.